service/docs/source/overview.rst

```diff
@@ -309,33 +312,31 @@ Reading Signals
 Understanding Aggregation
 """""""""""""""""""""""""
 
-The telemetry that is output from ``geopmread`` or the APIs will be
-automatically aggregated based on the requested domain and the aggregation
+The telemetry that is output from ``geopmread`` or the APIs will automatically
+be aggregated based on the requested domain and the aggregation
 type.
 
 Using ``CPU_FREQUENCY_STATUS`` as an example, the output  in `Listing Signal
 Information`_ shows the native domain as ``cpu`` and the aggregation type as
 ``average``.  Notice the :ref:`topology diagram <topo-diagram>` shows that CPUs
-are contained within cores, cores are contained within packages, and packages
-are contained within the board.
-
-If ``CPU_FREQUENCY_STATUS`` was requested at the ``core`` domain, all of the
-CPUs associated with that core will have their frequency read, and the results
-will be averaged together.  If the signal was requested at the ``package``
-domain, all of the CPUs associated with all of the cores on that package will
-have their frequency read, and the results will be averaged together.  This
-pattern continues up to the highest level domain: ``board``.  This means if
-you wanted to read the average frequency across all packages, cores, and CPUs
-in a system, you would issue a ``geopmread`` at the ``board`` domain.
+are contained within cores, cores within packages, and packages within the board.
+
+When a ``CPU_FREQUENCY_STATUS`` request is made at the ``core`` domain, GEOPM
+reads and averages the frequencies of all CPUs linked to that core. If the
+request is at the ``package`` domain, it aggregates the frequencies of all CPUs
+across every core in that package and provides the average. This methodology
+escalates up to the broadest domain, the ``board`` domain. Thus, to obtain the
+average frequency spanning all packages, cores, and CPUs in the system, one
+would issue a `geopmread` at the ``board`` domain.
 
 For more information about aggregation types, see: :doc:`GEOPM_CXX_MAN_Agg.3`.
 
 Reading Multiple Signals
 """"""""""""""""""""""""
-In order to regularly read platform telemetry for output directly to the
-console or to a file, use the ``geopmsession`` from the command-line (which takes
-input that is similar to the arguments of ``geopmread``) or the batch read API
-from code.
+To fetch platform telemetry and output it to the console or a file:
+
+- From the command-line: Use `geopmsession`. Its input arguments are similar to `geopmread`.
```

Comment:
> Its input arguments are similar to `geopmread`

I don't think that is true. The way `geopmsession` interprets `stdin` is similar to the way `geopmread` interprets its positional args.